introduction.md

```diff
@@ -1,3 +1,10 @@
 Hello, my name is Timothy Barnes. I am a mentor for Bloc.io. My shoes are off.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?